src/BcPrebidVast.js

```diff
@@ -0,0 +1,365 @@
+/**
+ * Header Bidding Plugin Brightcove module.
+ */
+
+// CONSTANTS
+
+var _vjs = window.videojs !== undefined ? window.videojs : null;
+var _prebidGlobal = require('./PrebidGlobal.js');
+var _vastManager = require('./VastManager.js');
+var _prebidCommunicator = require('./PrebidCommunicator.js');
+var _logger = require('./Logging.js');
+var _prefix = 'PrebidVast->';
+
+var $$PREBID_GLOBAL$$ = _prebidGlobal.getGlobal();
```

Comment:
are you using `$$PREBID_GLOBAL$$` as the actual symbol name or does this get replaced at build time like it does in Prebid.js ?

Reply:
It will be replaced during the build. I forgot do it in this commit. It will be done in next commit.